### ISSUE TITLE
(🐞) Bad `EmptyFunctionBlock` on empty function

### ISSUE THREAD

**Issue by KotlinIsland** [detekt/detekt#5273]
```kt
fun foo() {}  // This empty block of code can be removed. [EmptyFunctionBlock]

fun bar()  // Function 'bar' without a body must be abstract
```

## Expected Behavior
I'm not sure, no error I suppose, or a better suggestion.

## Observed Behavior
An error that when actioned, results in a compile error.

## Context / Real life

```kt
open class A {
    open fun foo() { println("hi") }
}
class B : A() {
    override fun foo() {}
}
```
## Your Environment

* Version of detekt used: 1.21.0
* Version of Gradle used (if applicable): 7.5.1
* Gradle scan link (add `--scan` option when running the gradle task): 
* Operating System and version: Windows 10
* Link to your project (if it's a public repository):

**Comment by schalkms** [comment]
This is not a bug. The rule works as intended. To get rid of the violation, you could either add a `//` comment in the function body or set the `ignoreOverridden` rule config to `true`.

I'm citing the rule documentation.

> [**EmptyFunctionBlock**​](https://detekt.dev/docs/rules/empty-blocks/#emptyfunctionblock)
> 
> Reports empty functions. Empty blocks of code serve no purpose and should be removed. This rule will not report functions with the override modifier that have a comment as their only body contents (e.g., a // no-op comment in an unused listener function).
> 
> Set the [ignoreOverridden] parameter to true to exclude all functions which are overriding other functions from the superclass or from an interface (i.e., functions declared with the override modifier).
